test/e2e/suites/e2e/e2e_test.go

```diff
@@ -208,8 +209,13 @@ var _ = Describe("e2e tests", func() {
 
 			shared.Byf("Creating MachineDeployment with custom port options")
 			md3Name := clusterName + "-md-3"
+
+			tTrue := true
 			customPortOptions := &[]infrav1.PortOpts{
-				{Description: "primary"},
+				{
+					Description: "primary",
+					Trunk:       &tTrue,
+				},
```

Comment:
Suggestion: we could have added a second port so the first would not be trunked and the second trunked.

Reply:
Thanks, will do that.

Reply:
Done, 
Tests have passed with the newly added trunked port as well.